vaadin-master-detail-layout-flow-parent/vaadin-master-detail-layout-flow/src/test/java/com/vaadin/flow/component/masterdetaillayout/MasterDetailLayoutTest.java

```diff
@@ -232,6 +232,71 @@ public void setDetailMinSizeWithUnit_getDetailMinSize() {
                 layout.getElement().getProperty("detailMinSize"));
     }
 
+    @Test
+    public void setOrientation_getOrientation() {
+        Assert.assertEquals(layout.getOrientation(),
+                MasterDetailLayout.Orientation.HORIZONTAL);
+
+        layout.setOrientation(MasterDetailLayout.Orientation.VERTICAL);
+
+        Assert.assertEquals(layout.getOrientation(),
+                MasterDetailLayout.Orientation.VERTICAL);
+    }
+
+    @Test
+    public void setContainment_getContainment() {
+        Assert.assertEquals(layout.getContainment(),
+                MasterDetailLayout.Containment.LAYOUT);
+
+        layout.setContainment(MasterDetailLayout.Containment.VIEWPORT);
+
+        Assert.assertEquals(layout.getContainment(),
+                MasterDetailLayout.Containment.VIEWPORT);
```

Comment:
Let's also test the element property value.

Reply:
Done.